command.go

```diff
@@ -93,7 +93,7 @@ func addCompletionCommand(targetCmd *cobra.Command) {
 	specCmd := &cobra.Command{
 		Use: "spec",
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Fprint(cmd.OutOrStdout(), spec.Spec(targetCmd))
+			fmt.Fprint(cmd.OutOrStdout(), spec.Spec(targetCmd)) // nolint:errcheck
```

Comment:
Configure errcheck instead of doing this

https://golangci-lint.run/usage/linters/#errcheck

Reply:
thanks for the heads up